cmd/hope/vm/image.go

```diff
@@ -55,17 +55,33 @@ var imageCmd = &cobra.Command{
 		}
 
 		for engine, engHypervisors := range hypEngMap {
-			log.Debugf("Creating VM %s using %d %s hypervisors", vm.Name, len(vm.Hypervisors), engine)
+			log.Debugf("Creating VM %s using %d %s hypervisors", vm.Name, len(engHypervisors), engine)
 			firstHV := engHypervisors[0]
+			remainingHVs := engHypervisors[1:]
 
-			packerSpec, err := firstHV.CreateImage(vms, *vm, *imageCmdParameterSlice, imageCmdForceFlag)
-			if err != nil {
+			if err := firstHV.CreateImage(vms, *vm, *imageCmdParameterSlice, imageCmdForceFlag); err != nil {
```

Comment:
I think this should be restructured to check the CopyImageMode first, and then make one call each to two functions

```
createImageFromHypervisors([]Hypervisor, VMs, VMImageSpec, []string bool)
copyImageToHypervisors([]Hypervisor, VMs, VMImageSpec, Hypervisor)
```

Something like that would avoid 2 places where images could be built, and 2 places where they can be copied.